librecad/src/actions/rs_actionpolylinesegment.cpp

```diff
@@ -54,9 +54,9 @@ void RS_ActionPolylineSegment::init(int status) {
 	if (container->countSelected(true, entityType)) {
 		//find a selected entity
 		//TODO, find a better starting point
-		for (RS_Entity* e = container->firstEntity(RS2::ResolveAllButInserts);
+		for (RS_Entity* e = container->firstEntity(RS2::ResolveNone);
```

Comment:
A polyline, in terms of DXF guidelines, is a list of points (vertices X,Y). Each vertex has a bulge parameter which describes the arc to the next vertex. A straight line is the special case when the bulge value is 0. Thus the DXF format isn't capable of elliptical arcs in polylines or 'child'-polylines.